Distribution/XmlSchemas/Readme.md

```diff
@@ -0,0 +1,90 @@
+    <DnaLibrary Name="Your Add-In" RuntimeVersion="v4.0" xmlns="http://schemas.excel-dna.net/addin/2018/05/dnalibrary">
+      <!-- (...) -->
+    </DnaLibrary>
+    ```
+
+### Install to a specific Excel-DNA project (manually)
+
+1. Download the XML Schema Definition files below to a folder on your computer:
+
+    * [`ExcelDna.DnaLibrary.xsd`][dna-xsd]
+    * [`customUI.xsd`][cui-2007-xsd]
+    * [`customui14.xsd`][cui-2010-xsd]
+
+2. Add these files to your Excel-DNA Add-In project or solution, so that Visual Studio can detect them to perform validation and provide IntelliSense to your `.dna` files and `.xml` files for Office Custom UI Ribbons, CTPs, etc.
+
+3. Add the XML namespace `http://schemas.excel-dna.net/addin/2018/05/dnalibrary` to your `.dna` file(s). E.g.:
+
+    ```xml
+    <?xml version="1.0" encoding="utf-8"?>
+    <DnaLibrary Name="Your Add-In" RuntimeVersion="v4.0" xmlns="http://schemas.excel-dna.net/addin/2018/05/dnalibrary">
+      <!-- (...) -->
+    </DnaLibrary>
+    ```
+
+### Install globally to all projects on your machine (via VSIX)
+
+(_coming soon_) follow this repository: <https://github.com/Excel-DNA/VSExcel>
+
+### Install globally to all projects on your machine (manually)
+
+This option requires you to copy files to Visual Studio's main installation folder, which means _you will need administrative privileges_ to perform get IntelliSense working with this approach.
+In the instructions below, `%ProgramFiles(x86)%` is used to refer to the `Program Files` folder. This will typically be `C:\Program Files` on 32-bit systems and `C:\Program Files (x86)` on 64-bit systems. You may need to alter the paths below if you installed Visual Studio in a different location.
+
+1. Download the XML Schema Definition files below to a folder on your computer:
+
+    * [`ExcelDna.DnaLibrary.xsd`][dna-xsd]
+    * [`customUI.xsd`][cui-2007-xsd]
+    * [`customui14.xsd`][cui-2010-xsd]
+    * [`ExcelDnaCatalog.xml`][dna-catalog-xml]
+
+2. Copy these files to your the Visual Studio Global Schema Cache. This will be `%ProgramFiles(x86)%\Microsoft Visual Studio [xx]\Xml\Schemas\` for Visual Studio 2015 and earlier or `%ProgramFiles(x86)%\Microsoft Visual Studio\[xx]\[edition]\Xml\Schemas\` for Visual Studio 2017 and later. `[xx]` will be the version such as `14.0` or `2017` and `[edition]` will be the edition such as `Community`, `Professional`, or `Enterprise`.
+
+    For more detailed information about [Visual Studio's Schema Cache][vs-schema-cache], see the documentation on Microsoft's website: [https://docs.microsoft.com/en-us/visualstudio/xml-tools/schema-cache][vs-schema-cache]
+
+3. Restart any open instances of Visual Studio. IntelliSense and validation should now work for all `.dna` files you open, automatically, even if they don't have the XML namespace for `.dna` files (`http://schemas.excel-dna.net/addin/2018/05/dnalibrary`) declared (although it a good practice to always include the XML namespace on your files).
```

Comment:
This is not needed.

Reply:
Hmm. Are you sure? It didn't work for me on FSharp projects until I restarted VS. I'll reword it as to "try it, and if doesn't work, restart" kind of thing